TransformationSystem/Client/TransformationClient.py

```diff
@@ -179,62 +200,65 @@ def getTransformationTasks( self, condDict = None, older = None, newer = None, t
     while True:
-      res = rpcClient.getTransformationTasks( condDict, older, newer, timeStamp, orderAttribute, limit,
-                                              inputVector, offsetToApply )
+      res = rpcClient.getTransformationTasks(condDict, older, newer, timeStamp, orderAttribute, limit,
+                                             inputVector, offsetToApply)
       if not res['OK']:
         return res
       else:
-        gLogger.verbose( "Result for limit %d, offset %d: %d" % ( limit, offsetToApply, len( res['Value'] ) ) )
+        gLogger.verbose("Result for limit %d, offset %d: %d" % (limit, offsetToApply, len(res['Value'])))
         if res['Value']:
           transformationTasks = transformationTasks + res['Value']
           offsetToApply += limit
-        if len( res['Value'] ) < limit:
+        if len(res['Value']) < limit:
           break
-    return S_OK( transformationTasks )
+    return S_OK(transformationTasks)
 
-  def cleanTransformation( self, transID ):
+  def cleanTransformation(self, transID):
     """ Clean the transformation, and set the status parameter (doing it here, for easier extensibility)
     """
     # Cleaning
     rpcClient = self._getRPC()
-    res = rpcClient.cleanTransformation( transID )
+    res = rpcClient.cleanTransformation(transID)
     if not res['OK']:
       return res
     # Setting the status
-    return self.setTransformationParameter( transID, 'Status', 'TransformationCleaned' )
+    return self.setTransformationParameter(transID, 'Status', 'TransformationCleaned')
 
-  def moveFilesToDerivedTransformation( self, transDict, resetUnused = True ):
+  def moveFilesToDerivedTransformation(self, transDict, resetUnused=True):
     """ move files input to a transformation, to the derived one
     """
     prod = transDict['TransformationID']
-    parentProd = int( transDict.get( 'InheritedFrom', 0 ) )
+    parentProd = int(transDict.get('InheritedFrom', 0))
     movedFiles = {}
     if not parentProd:
-      gLogger.warn( "[None] [%d] .moveFilesToDerivedTransformation: Transformation was not derived..." % prod )
-      return S_OK( ( parentProd, movedFiles ) )
+      gLogger.warn("[None] [%d] .moveFilesToDerivedTransformation: Transformation was not derived..." % prod)
+      return S_OK((parentProd, movedFiles))
     # get the lfns in status Unused/MaxReset of the parent production
-    res = self.getTransformationFiles( condDict = {'TransformationID': parentProd, 'Status': [ 'Unused', 'MaxReset' ]} )
+    res = self.getTransformationFiles(condDict={'TransformationID': parentProd, 'Status': ['Unused', 'MaxReset']})
     if not res['OK']:
-      gLogger.error( "[None] [%d] .moveFilesToDerivedTransformation: Error getting Unused files from transformation %s:" % ( prod, parentProd ), res['Message'] )
+      gLogger.error("[None] [%d] .moveFilesToDerivedTransformation:"
+                    " Error getting Unused files from transformation %s:" % (prod, parentProd), res['Message'])
```

Comment:
No, this should be:
```suggestion
gLogger.error("[None] [%d] .moveFilesToDerivedTransformation:" % prod,
                      " Error getting Unused files from transformation %s: %s " % (parentProd, res['Message']))
```
and also the next ones should be corrected.

Reply:
OK, will do.. This is not my code but happy to fix it!

Reply:
Hmmm.... it's not even correct as the first part should in principle not contain the transformation ID... This message structure `[thread] [transID] method:` is used at many many places and is incompatible with fixed part and variable part... 
What would you suggest?

Reply:
We miss a prefix in the `gLogger` messages, but this can be done with a sub logger... I can try and do that?

Reply:
It's not compatible with the fixed and variable part, if this can be done with a sub logger, why not? Because as you say it should be in the prefix part.

Reply:
Is it better now?

Reply:
Looks better. We actually have the ~same thing in `TransformationAgentUtilities.py`, in case you are feeling up to it...

Reply:
Why not, while being confined... I do it quickly as in an hour I am planning to mow the lawn! ;-)